test/unit/range/container/container_test_template.hpp

```diff
@@ -20,21 +20,17 @@
 using seqan3::operator""_dna4;
 
 template <typename T>
-class container_ : public ::testing::Test
+class container_over_dna4_test : public ::testing::Test
 {};
 
-using container_types = ::testing::Types<std::vector<seqan3::dna4>,
-                                         seqan3::bitcompressed_vector<seqan3::dna4>,
-                                         seqan3::small_vector<seqan3::dna4, 1000>>;
```

Comment:
Here you remove the tests for 3 different types, but you only add the test for bitcompressed again. Shouldn't we keep tests also for small_vector and vector?

Reply:
I also added the test template in small_vector_test.cpp !
And I thought we do no need to test `std::vector` :woman_shrugging:

Reply:
True... sorry I couldn't find it, but I see it's there.